### PR TITLE
Created Ui/frontend elements for User story 6

### PR DESCRIPTION
**Pull request by MosesAfrane**

### CODE REVIEW

File: src/components/addTask.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    DESCRIPTION: eventDescription.value,
+    SEQUENCE: 0,
+    SUMMARY: eventName.value,
+    URL: "",
+    Submit: submitting,
+  };
+
+  /* Sending to backend */
+  axios
+    .post("http://localhost:5000/events/add", {
```
Comment (daviddecorso) [inline src/components/addTask.js:44]: add line to send user id: `userID: user._id`

File: src/components/addTask.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
+import React, { useState } from "react";
+import DateTimePicker from 'react-datetime-picker';
+import axios from "axios";
+
+
+function AddTask (){
```
Comment (daviddecorso) [inline src/components/addTask.js:6]: pass user props from homepage to taskview to addtask

File: src/components/addTask.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+              </div>
+              <p class = "helpDescription"> </p>
+            </div>
+          </form>
+          <form>
+            <div class = "field">
+              <label class = "label"> Event Date</label>
+              <form>
+           <section class = "field mb-6">
+           <DateTimePicker
```
Comment (daviddecorso) [inline src/components/addTask.js:129]: change styling to be more readable

File: src/components/addTask.js [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        </button>
+      </div>
+     <div class = "modal"
+       className = "modal"
+       id = 'modal'>
+        <div class = "modal-background"
+           id = 'modalbg' >
+        </div>
+        <button onClick = {onClickExit}
+              class= "modal-close is-large"
```
Comment (daviddecorso) [inline src/components/addTask.js:88]: close button a bit far from task card

**Review by daviddecorso — DISMISSED** [review]

**Review by lucasrossell — APPROVED** [review]
lgtm. add task button confirmed working on front end + back end

**Review by lucasrossell — APPROVED** [review]
yep

**Pull request merged**